docker/Dockerfile.test_brute

```diff
@@ -0,0 +1,11 @@
+FROM daq/aardvark:latest
+
+RUN $AG update && $AG install python
+
+COPY subset/pentests/user_list.json .
+COPY subset/pentests/test_brute .
+COPY subset/pentests/brute_client.py .
+
+#CMD python client.py $TARGET_IP 10000
```

Comment:
??

Reply:
removed, originally was there for a pointer to my internal memory to remind me that I could run test directly from Dockerfile